internal/user/user_manager.go

```diff
@@ -21,6 +21,9 @@ const UsernameAtHostnameLabel = "username"
 // ErrUserNotFound is returned to indicate that a user was not found in the system.
 var ErrUserNotFound = errors.New("user not found")
 
+// ErrUserAlreadyExists indicates that a user already exist in the system when attempting to create a new one.
+var ErrUserAlreadyExists = errors.New("user already exists found")
```

Comment:
Was this meant to be just `"user already exists"`?
(`"already exists found"` sounds kinda weird)

Reply:
yeap, probably edit error.